Realm/Realm/Native/NativeCommon.cs

```diff
@@ -21,38 +21,15 @@
 using System.IO;
 using System.Reflection;
 using System.Runtime.InteropServices;
-using System.Text;
 using System.Threading;
-using Realms.Native;
 
 namespace Realms
 {
     internal static class NativeCommon
     {
-#if DEBUG
-        [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
-        public unsafe delegate void DebugLoggerCallback(byte* utf8String, IntPtr stringLen);
-
-        [MonoPInvokeCallback(typeof(DebugLoggerCallback))]
-        private static unsafe void DebugLogger(byte* utf8String, IntPtr stringLen)
-        {
-            var message = Encoding.UTF8.GetString(utf8String, (int)stringLen);
-            Console.WriteLine(message);
-        }
-
-        [DllImport(InteropConfig.DLL_NAME, EntryPoint = "set_debug_logger", CallingConvention = CallingConvention.Cdecl)]
-        public static extern void set_debug_logger(DebugLoggerCallback callback);
-#endif  // DEBUG
-
         [DllImport(InteropConfig.DLL_NAME, EntryPoint = "delete_pointer", CallingConvention = CallingConvention.Cdecl)]
         public static extern unsafe void delete_pointer(void* pointer);
 
-        [DllImport(InteropConfig.DLL_NAME, EntryPoint = "delete_pointer", CallingConvention = CallingConvention.Cdecl)]
-        public static extern unsafe void delete_pointer(IntPtr pointer);
-
-        [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_reset_for_testing", CallingConvention = CallingConvention.Cdecl)]
-        public static extern void reset_for_testing();
```

Comment:
Should this deletion and the one above part of this PR?

Reply:
I was cleaning up the debug logger stuff and noticed those are not used, which allowed me to remove debug.cpp.